data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals VALUES (1, 'Agumon', '2020/02/03', 0, false, 10.23);
+INSERT INTO animals VALUES (2, 'Gabumon', '2018/11/15', 2, true, 8);
+INSERT INTO animals VALUES (3, 'Pikachu', '2021/01/07', 1, false, 15.04);
+INSERT INTO animals VALUES (4, 'Devimon', '2017/05/12', 5, true, 11);
```

Comment:
- [ ] when inserting into the table you need to specify which column as well like:
```
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Agumon', '2020/2/3', '0', 'true', '10.23');
```

Reply:
Hi @mikethreels, I appreciate the feedback, however I don't need to specify wich column as long as I write the values in order, Is there any consideration I need to have in mind that make specify the column imperative?
Thank you for taking the time to answer my question.